examples/src/main/res/values/strings.xml

```diff
@@ -58,6 +58,9 @@
     <string name="title_guidance_view">GuidanceView</string>
     <string name="description_guidance_view">Demonstrates the use of a GuidanceView to show an instructional image at proper junctions.</string>
 
+    <string name="title_junction_snapshot_sample" translatable="false">Snapshotter Sample(POC)</string>
+    <string name="description_junction_snapshot_sample" translatable="false">Demonstrates junction view snapshot.</string>
```

Comment:
This shouldn't be `translatable = "false"`. Actually none of the resources here shouldn't 👀 https://github.com/mapbox/mapbox-navigation-android/pull/3438#issue-468106657 https://github.com/mapbox/mapbox-navigation-android/issues/2909 https://github.com/mapbox/mapbox-navigation-android/pull/3425#discussion_r471812462